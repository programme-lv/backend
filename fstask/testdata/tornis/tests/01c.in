9
4 6 5 7 9 8 2 1 3
