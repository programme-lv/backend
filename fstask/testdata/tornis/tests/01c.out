9 8
5 6
4 5
4 7
8 9
4 8
1 2
1 3
1 4